src/cli/doc_get.rs

```diff
@@ -274,7 +274,7 @@ pub(crate) fn ids_from_input(
 }
 
 #[derive(Debug)]
-struct GetResult {
+pub struct GetResult {
```

Comment:
Nit: Should probably be pub(crate) but doesn't matter enough to block merging.

Reply:
Will change, then merge after building